spec/controllers/admin/principals_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+RSpec.describe Admin::PrincipalsController, type: :controller do
+  before { sign_in(user) }
+
+  let(:user) { FactoryGirl.create(:user, principal: principal) }
```

Comment:
Is this an opportunity to upgrade to FactoryBot?

Reply:
@margOnline good point, unfortunately I have a PR that is a bit chunky on the way, so it would cause me some rebase issues I'd rather avoid now, definitely worth adding a card to the tech debt backlog though.